code/src/styles/styles.scss

```diff
@@ -0,0 +1,477 @@
+    color: $cream;
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  &__image {
+    height: 100%;
+    filter: invert(1);
+    color: #000;
+  }
+
+  &__name {
+    font-size: 15px;
+    text-align: right;
+  }
+
+  @include respond(medium) {
+    &__nav {
+      display: flex;
+    }
+
+    &__nav--open {
+      all: unset;
+      display: flex;
+    }
+
+    &__nav--closed {
+      display: flex;
+    }
+
+    &__button {
+      display: none;
+    }
+
+    &__name {
+      font-size: 20px;
+    }
+  }
+}
+
+/*--- List Grid ---*/
+
+.list {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  position: relative;
+  width: 100%;
+
+  @include respond(medium) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+/*--- List Movie Thumb ---*/
+
+.movie-thumb {
+  position: relative;
+  text-align: center;
+
+  &__link {
+    color: $cream;
+    text-decoration: none;
+    font-weight: bold;
+    display: block;
+  }
+
+  &__image {
+    width: 100%;
+  }
+
+  &__overlay {
+    display: none;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background: linear-gradient(rgba($dark-blue, 0.2) 0, rgb(21, 23, 61) 80%);
+    padding: 40px 10px;
+    transition: all 0.5s ease-in-out;
+  }
+
+  &__title {
+    font-size: 35px;
+    margin-bottom: 10px;
+  }
+
+  &:hover .movie-thumb__overlay {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+}
+
+/*** MOVIE DETAIL PAGE ***/
+
+.detail {
+  display: grid;
+  min-height: 100vh;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  color: $cream;
+
+  &__info-container {
+    display: flex;
+    align-items: flex-end;
+    padding: 5%;
+  }
+
+  &__image {
+    display: none;
+    margin-right: 20px;
+    border: 2px solid $cream;
+
+    @include respond(medium) {
+      display: flex;
+    }
+  }
+
+  &__title {
+    text-transform: uppercase;
+    margin-bottom: 0px;
+    text-decoration: none;
+    color: $cream;
+    transition: 0.2s ease-in-out;
+
+    &:hover {
+      text-decoration: underline;
+      transform: scale(1.01);
+    }
+  }
+
+  &__star {
+    color: $beige;
+    font-size: 20px;
+  }
+
+  &__vote {
+    font-size: 16px;
+    font-weight: 600;
+    margin-left: 5px;
+    color: red;
+  }
+
+  &__tagline {
+    font-style: italic;
+  }
+
+  &__overview {
+    font-family: 'Montserrat', sans-serif;
+  }
+
+  &__logo {
+    color: $beige;
+    font-size: 35px;
+    background: transparent;
+    transition: 0.2s ease;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+  }
+
+  &__subheader {
+    margin: 15px 0 5px 0;
+    font-weight: 600;
+    text-transform: uppercase;
+    letter-spacing: 0.2em;
+  }
+
+  &__company-container {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  &__company {
+    color: #15173d;
+    text-decoration: none;
+    padding: 2px 5px;
+    background: $turquoise;
+    font-size: 14px;
```

Comment:
This was a bit difficult to read on the screen. Maybe you could increase the font-size a bit.